src/sinol_make/__init__.py

```diff
@@ -30,7 +32,7 @@ def configure_parsers():
     return parser
 
 
-def main():
+def _main_exn():
```

Comment:
```suggestion
def main_exn():
```
I don't see any reason to hide this function now that its name describes what it does.